### DISCUSSION TITLE
Cropping videos

### DISCUSSION THREAD

**Discussion by willjoe** [sorayuki/obs-multi-rtmp#98] [Ideas]
Thank you very much for you great plugin. ありがとうございます。

I was wondering if there can be a cropping feature where we can change the aspect ratio per destination.
Youtube currently only supports 16:9, but facebook live can support many kinds of aspect ratio.
I would like to crop my facebook live video to 4:3.

Again, thank you!

**Comment by sorayuki** [comment]
Hi, currently the only way I have found to output in different video size in OBS's standard render pipeline is to scale video, not crop.
